CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 0.1.1
 =====
 ## [Bugfix release v0.1.1](https://github.com/rogalmic/vscode-bash-debug/releases/tag/v0.1.1)
+- new launch parameters: program, args, cwd, pathBash, pathBashdb, pathCat, pathMkfifo, pathPkill, trace
+- [BREAKING]deprecated parameters: scriptPath -> program, commandLineArguments -> args, bashDbPath -> pathBashdb, bashPath -> pathBash
 - fix default launch.json generation after breaking change in Microsoft's debug adapter interface
-- various fixes
+- fix pkill usage on macOS
```

Comment:
Could you remove this line for now?
I want write one more fix for pkill path handling on macOS.